.evergreen/start-orchestration.sh

```diff
@@ -67,17 +67,21 @@ killport() {
 
   if [[ "${OSTYPE:?}" == cygwin || "${OSTYPE:?}" == msys ]]; then
     for pid in $(netstat -ano | grep ":$port .* LISTENING" | awk '{print $5}' | tr -d '[:space:]'); do
-      taskkill /F /T /PID "$pid" || true
```

Comment:
`/F` already force kills so we shouldn't need to wait here.

Reply:
This PR proposes using `kill` instead of `taskkill` to extend graceful termination + timeout->kill to Windows distros as well. Please re-review.

Reply:
Should we revert this now as well?

Reply:
I am in favor of keeping this change as a simplification + consistency improvement given this script is known to be Bash-executed. The less distro-specific patterns we need to depend on, the better.

Reply:
I'm concerned about this one. `taskkill /F /T ` kills the process and all child processes which seems safer than just `kill -SIGKILL`. I don't want to make windows less stable after this change so I'd prefer we keep it using taskkill

Reply:
Done.